code-studio/src/js/components/progress/course_progress.jsx

```diff
@@ -0,0 +1,42 @@
+/* global React, dashboard */
+
+window.dashboard = window.dashboard || {};
+
+/**
+ * Stage progress component used in level header and course overview.
+ */
+window.dashboard.CourseProgress = (function (React) {
```

Comment:
I think this will still be needed for non-code-studio files to reference the component.  For example, Dashboard or Pegasus inline JS.